stat.go

```diff
@@ -648,6 +648,71 @@ func KullbackLeibler(p, q []float64) float64 {
 	return kl
 }
 
+// OrdinaryLinearRegression computes the simple linear regression of x and y.
+// It returns the values for \hat{\alpha}, \hat{\beta} and the coefficient
```

Comment:
We should write out the parameters in the equation. There was a really cool map I saw at one point that showed which constants various parts of the world learned (I learned it as m*x + b).